site/_apps/compost-hn.txt

```diff
@@ -0,0 +1,23 @@
+---
+weight: 31
+title: 'Vanilla Web Components with Compost mixins'
+github-title: 'lamplightdev/compost-hn'
+library: 'Compost mixins'
+module-bundling: 'Rollup'
+service-worker: 'Shell + data caching with Workbox'
+perfomance-patterns: 'Local caching of API results, CDN cache'
+server-side-rendering: 'no'
+api: 'Node-hnapi (unofficial)'
```

Comment:
@davideast rolled out a faster API that you can use here with a CDN cache (api.hnpwa.com/v0) so I would definitely consider using that here instead!

Reply:
Yes! I would love to see the results. Check it out here: https://blog.hnpwa.com/introducing-the-hnpwa-api-656db6b88ec1